### PR TITLE
Improve keyboard control for survey task choices

### PR DESCRIPTION
**Pull request by eatyourgreens**
Towards #2132 

Describe your changes.
Autofocus the survey image flipper when the choice first mounts, allowing images to be changed with the cursor keys.
Autofocus the confusions popup when that opens, so that it can be dismissed from the keyboard if required.
Add a visual style to question answers when those radio buttons have focus.

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch? https://survey-choice-keyboard.pfe-preview.zooniverse.org/
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
The Choice component only receives focus if it has an image slideshow, but this seems to be a step in the right direction.

**Comment by eatyourgreens** [comment]
https://survey-choice-keyboard.pfe-preview.zooniverse.org/projects/pat-lorch/focus-on-wildlife-cleveland-metroparks/classify?env=production seems to be a good project to test this out. Open a choice, then navigate through it with the cursor and tab keys.

File: app/classifier/tasks/survey/choice.cjsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ module.exports = React.createClass
                   <ImageFlipper images={@props.task.images[filename] for filename in otherChoice.images} />
                   <Markdown content={choice.confusions[otherChoiceID]} />
                   <div className="survey-task-choice-confusion-buttons" style={textAlign: 'center'}>
-                    <button type="submit" className="major-button identfiy">Dismiss</button>
+                    <button type="submit" autoFocus={true and otherChoice.images.length < 2} className="major-button identfiy">Dismiss</button>
```
Comment (mcbouslog) [inline app/classifier/tasks/survey/choice.cjsx]: this is just for my understanding - but why the `true and ` before the condition we're interested in?

Comment (eatyourgreens) [inline app/classifier/tasks/survey/choice.cjsx]: Ha! Nothing clever. This is a mistake on my part, not reading the code properly.

Comment (eatyourgreens) [inline app/classifier/tasks/survey/choice.cjsx]: I'm going to have to revisit the logic for how autofocus is assigned when the image slideshow isn't present, so I'll leave changing this to a future PR.

**Pull request merged**